exist-distribution/pom.xml

```diff
@@ -1057,7 +1057,7 @@
                             <identifier>org.exist.start.Main</identifier>
                             <mainClassName>org.exist.start.Main</mainClassName>
                             <executableName>eXist-JavaAppLauncher</executableName>
-                            <jvmRequired>1.8</jvmRequired>
+                            <jvmRequired>21</jvmRequired>
```

Comment:
@duncdrum @line-o this might be an important one

Reply:
No immediate worries from my side. Looking good